src/matrix_graph.rs

```diff
@@ -169,12 +170,27 @@ not_zero_impls!(u8, u16, u32, u64, usize);
 not_zero_impls!(i8, i16, i32, i64, isize);
 not_zero_impls!(f32, f64);
 
+/// Short version of `NodeIndex::new`
 #[inline]
 pub fn node_index(ax: usize) -> NodeIndex {
     debug_assert!(ax.index() <= Ix::max_value() as usize);
     NodeIndex::new(ax)
 }
 
+/// `MatrixGraph<N, E, Ty, Null>` is a graph datastructure using an adjacency matrix
+/// representation.
+///
+/// `MatrixGraph` is parameterized over:
+///
+/// - Associated data `N` for nodes and `E` for edges, called *weights*.
+///   The associated data can be of arbitrary type.
+/// - Edge type `Ty` that determines whether the graph edges are directed or undirected.
+/// - Nullable type `Null`, which denotes the edges' presence (defaults to `Option<E>`). You may
+///   specify `NotZero<E>` if you want to use a sentinel value (such as 0) to mark the absence of
```

Comment:
Could you add a link to `NotZero` docs here?

Reply:
You did link this in the next commit, sorry. But the other changes are still valid.